requirements.txt

```diff
@@ -14,5 +14,4 @@ psycopg2-binary
 pandas
 argschema==2.0.1
 jsonlines
-opencv-python
```

Comment:
this is still a requirement:
```
(label) danielk@ibs-danielk-ux2:~/segmentation-labeling-app$ grep -rnw ./ -e "import cv2"
./tests/test_transformations.py:5:import cv2
./segmentation_labeling_app/rois/rois.py:6:import cv2
```

Reply:
You can't see it in this file snippet, but it was listed twice

Reply:
It's also on line 3 if you click the full file

Reply:
oh, thanks, really had to open that file to see that.